src/elements/CommentCard.jsx

```diff
@@ -0,0 +1,13 @@
+export const CommentCard = ({ comment }) => {
+  console.log(comment);
+  return (
+    <li Key={comment.comment_id} className="commentCard">
+      {comment.author} said:{" "}
```

Comment:
the way this is laid out at the moment isn't super clear, the date comes right after said: as opposed to the comment, but it's a minor issue